operator/autopilot-channels/packages/configconnector/1.134.0/cluster/gcp-identity/0-cnrm-system.yaml

```diff
@@ -0,0 +1,3187 @@
+apiVersion: v1
```

Comment:
Would be better if we were diffing against the previous version for these files.